tests/test_invites.py

```diff
@@ -37,7 +35,7 @@ def test_delete_on_bind(self):
         def post_json_get_nothing(uri, post_json, opts):
             return Response((b"HTTP", 1, 1), 200, b"OK", None, None)
 
-        FederationHttpClient.post_json_get_nothing = Mock(
+        FederationHttpClient.post_json_get_nothing = AsyncMock(
```

Comment:
Instead of using an `AsyncMock` here it is probably clearer to leave it as a `Mock`, but update `def post_json_get_nothing` above it to `async def post_json_get_nothing`.

(The same goes for below.)